torch/_export/serde/serialize.py

```diff
@@ -252,17 +252,21 @@ def deserialize_meta_func(serialized_target: str):
 def serialize_operator(target) -> str:
     if isinstance(target, str):
         return target
-    return f"{target.__module__}.{target.__name__}"
+    elif target.__module__.startswith("torch._ops"):  # From torch.fx.node._get_qualified_name
+        module = target.__module__.replace("torch._ops", "torch.ops")
+        return f"{module}.{target.__name__}"
+    else:  # TODO(zhxchen17) Don't catch all here.
+        return f"{target.__module__}.{target.__name__}"
```

Comment:
Do you want to replace this with a call to this function: https://www.internalfb.com/code/fbsource/[de0be07c12d0]/fbcode/caffe2/torch/fx/node.py?lines=80 ?

Reply:
I can leave a TODO here.